pkg/virt-operator/resource/generate/components/scc.go

```diff
@@ -74,7 +74,7 @@ func NewKubeVirtControllerSCC(namespace string) *secv1.SecurityContextConstraint
 	scc.SELinuxContext = secv1.SELinuxContextStrategyOptions{
 		Type: secv1.SELinuxStrategyRunAsAny,
 	}
-	scc.AllowedCapabilities = []corev1.Capability{"NET_ADMIN", "SYS_NICE"}
+	scc.AllowedCapabilities = []corev1.Capability{"NET_ADMIN", "SYS_NICE", "SYS_RESOURCE"}
```

Comment:
Please update the relevant [test](https://github.com/kubevirt/kubevirt/blob/2984c5befe204d62f5e73ca06dff25675aeea58e/tests/security_features_test.go#L244).

Reply:
Moving those tests from `vmi_multus_test` to another test suite
requires more changes such as exporting [common functions](https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_multus_test.go#L634-L665) to create SRIOV VM's to another package,
change [automation/test.sh](https://github.com/kubevirt/kubevirt/blob/master/automation/test.sh#L293-L294) so it could run tests from different test suit,  these are the changes I know off.
I prefer to keep those tests at `vmi_multus_test` for now as we have all SRIOV related tests in this test suite, and do that in a following PR.